lib/Lifters/BasicBlockLifter.cpp

```diff
@@ -42,7 +42,6 @@ CallableBasicBlockFunction BasicBlockLifter::LiftBasicBlockFunction() && {
 
 
   this->RecursivelyInlineFunctionCallees(bbfunc.func);
-  anvill::EntityLifter lifter(options);
```

Comment:
This constructor loads the semantic module for the given architecture and is responsible for a good chunk of the runtime. I'm assuming since `lifter` doesn't get used, we can just remove this.